app/src/main/java/io/homeassistant/companion/android/assist/AssistActivity.kt

```diff
@@ -129,9 +132,9 @@ class AssistActivity : BaseActivity() {
         viewModel.setPermissionInfo(hasRecordingPermission()) { requestPermission.launch(Manifest.permission.RECORD_AUDIO) }
     }
 
-    override fun onPause() {
-        super.onPause()
-        viewModel.onPause()
+    override fun onDestroy() {
```

Comment:
I can understand changing this to onDestroy for audio playback, but at least for recording it was intentionally on onPause.

A lot of people press home instead of back/finish when exiting apps, and we don't want to have to wait for the timeout for the mic (which can take a while, on server signal and it is a bit slow with it) to stop recording.

Reply:
Makes sense; adding `viewModel.onPause` back with just `stopRecording()`